100_essentials/greetings/README.md

```diff
@@ -0,0 +1,59 @@
+# Greetings
+
+Everything you need to complete this challenge can be found [here](https://docs.python.org/3.8/tutorial/controlflow.html#defining-functions).
```

Comment:
I like this as a pattern for 100-level challenges. It's a good way to ensure that they have the documentation they need and it _should_ at least help people learn to read and reference the official Python documentation.